src/FSharp.Core/reflect.fs

```diff
@@ -1193,7 +1235,7 @@ type FSharpType =
         then
             invalidArg "types" (SR.GetString(SR.nullsNotAllowedInArray))
 
-        mkTupleType false asm types
+        ObsoleteTupleTypeMaking.mkTupleType false asm types
```

Comment:
Really obsolete?

Reply:
From the user's perspective as well as ours, it is - noone is supposed to be using it directly.